blocks/init/src/Blocks/custom/image/manifest.json

```diff
@@ -29,5 +29,10 @@
 	},
 	"components": {
 		"image": "image"
+	},
+	"tailwind": {
+		"base": {
```

Comment:
This should be a `part`

Reply:
I can make another PR for this release to sort that out elsewhere, then it would make sense to update it here as well

Reply:
@goranalkovic-infinum here's the #59 😉